core/lib/types/src/api/mod.rs

```diff
@@ -443,6 +446,9 @@ pub struct Log {
     pub log_type: Option<String>,
     /// Removed
     pub removed: Option<bool>,
+    /// L2 block timestamp
+    #[serde(rename = "blockTimestamp")]
+    pub block_timestamp: Option<i64>,
```

Comment:
If I'm not mistaken adding `block_timestamp` to receipt is not a part of proposal. You can keep timestamp in `StorageTransactionReceipt`, it should be enough

Reply:
good point - skipped serialisation for this field. I'd prefer to leave the field in the struct though because that'd allow to keep `get_transaction_receipts` logic without major changes.

Reply:
But the serialization is the main purpose of this struct, I insist on removing `block_timestamp` from it

Reply:
fair enough - removed the field and modified the mapping function